news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,173 @@
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/every.jpg)](https://osu.ppy.sh/beatmapsets/905599)
+
+If you had to describe this map with one word, it'd be something along the lines of... Intricate. Detailed. Accurate. Clean. Recently we have seen a rise of mappers in osu!mania with this approach in mind, [Monheim](https://osu.ppy.sh/users/6814203) and [Vortex-](https://osu.ppy.sh/users/4999669) being well-known for their precision and accuracy in their maps. This map in particular accomplishes things that many mappers most likely would not be able to accomplish if they were to map this - it is musically accurate in that it expresses many instruments and melody lines at once, is delicate and clean to play, and provides a challenge for all sorts of players. These remain consistent through each difficulty of the mapset. This map thus deserves its spot in this season's spotlights, and be on the lookout - we most definitely will see more by these two extraordinarily talented mappers in the future.
+
+*written by [-mint-](https://osu.ppy.sh/users/8976576)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/oceanus.jpg)](https://osu.ppy.sh/beatmapsets/539179)
+
+It has been a while before [Level 51](https://osu.ppy.sh/users/3617847) made his appearance in mapping ground, and this time he brings the community with a great 6k mapset, Oceanus. GALE_FORCE, as the top difficulty of the mapset, proposes a real challenge for advanced players with its dense but well-balanced streams that follow the shifts in music. On the other hand, the lower two difficulties definitely provide great opportunities for general players to start with 6K and practice over 6K streams as the mapper managed to effectively balance the density within and between these difficulties. Hitsounds are designated in a way that reasonably support musical rhythms but do not distract players from focusing on the streams. That is to say, Oceanus deserves a place in spotlight as a recommendation to every player interested in progressing and challenging with 6K maps.
+
+*written by [Shima Rin](https://osu.ppy.sh/users/6089608)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/altros.jpg)](https://osu.ppy.sh/beatmapsets/877895)
+
+It has been about 3 years since [ByeForNow](https://osu.ppy.sh/users/7199159) has had a ranked map, and he comes back with an absolutely spectacular set! Despite being only three difficulties, each difficulty manages to have its own solid structure where instruments are represented through columns instead of the usual chords. You might want to turn on the hitsounds as well because the set is beautifully keysounded. Although keysounded, the set has high focus on the overall pitch and ByeForNow nails it perfectly! Delicate and precise, ByeForNow's Altros deserves a spot in this season's spotlights, and we will hope to see what other fantastic maps he produces!
```

Comment:
```suggestion
It has been about 3 years since [ByeForNow](https://osu.ppy.sh/users/7199159) has had a ranked map, and he comes back with an absolutely spectacular set! 

Despite being only three difficulties, each difficulty manages to have its own solid structure where instruments are represented through columns instead of the usual chords. You might want to turn on the hitsounds as well because the set is beautifully keysounded. Although keysounded, the set has a high focus on the overall pitch and ByeForNow nails it perfectly! 

Delicate and precise, ByeForNow's Altros deserves a spot in this season's Spotlights, and we will hope to see what other fantastic maps he produces!
```